Ofqual.Recognition.Citizen.API.Core/Models/DTO/TaskSectionDto.cs

```diff
@@ -0,0 +1,9 @@
+namespace Ofqual.Recognition.Citizen.API.Core.Models;
+
+public class TaskSectionDto
```

Comment:
So, this is probably where you need to be defining interfaces to be able to connect the different DTOs and original models together (and it'll mean that you get consistency across them all!)

For these, I would define:

- An interface representing the base members required for a TaskStatus (ITaskStatus)
- An interface representing the base members required for a Section (ISection)

And then put these interfaces in the definitions of the models. You can then start to make use of this in the code; as an example, on line 8 in this DTO, you can replace the List<TaskStatusDto> with List<ITaskStatus>.